deploy/cr-trial.yaml

```diff
@@ -4,20 +4,42 @@ metadata:
   name: trial
 spec:
   environment: trial
-  console:
-    resources:
-      requests:
-        memory: 1Gi
-      limits:
-        memory: 2Gi
-    env:
-    - name: key1
-      value: value1
-    - name: key2
-      value: value2
-  server:
-    resources:
-      requests:
-        memory:
-      limits:
-        memory: 2Gi
+  objects:
+    console:
+      resources:
+        limits:
+          memory: 2Gi
+      env:
+      - name: KIE_ADMIN_USER
+        value: adminUser
+      - name: KIE_ADMIN_PWD
+        value: RedHat
+      - name: KIE_SERVER_CONTROLLER_USER
+        value: controllerUser
+      - name: KIE_SERVER_CONTROLLER_PWD
+        value: RedHat
+      - name: KIE_SERVER_USER
+        value: executionUser
+      - name: KIE_SERVER_PWD
+        value: RedHat
+      - name: KIE_MAVEN_PWD
+        value: RedHat
+    server:
+      resources:
+        requests:
+          memory:
+      env:
+      - name: KIE_ADMIN_USER
+        value: adminUser
+      - name: KIE_ADMIN_PWD
+        value: RedHat
+      - name: KIE_SERVER_CONTROLLER_USER
+        value: controllerUser
+      - name: KIE_SERVER_CONTROLLER_PWD
+        value: RedHat
+      - name: KIE_SERVER_USER
+        value: executionUser
+      - name: KIE_SERVER_PWD
+        value: RedHat
+      - name: RHPAMCENTR_MAVEN_REPO_PASSWORD
+        value: RedHat
```

Comment:
Should be in the env file, not the CR. The CR should be empty to get the default environment behavior, and would provide passwords only to override what's the default. In fact, we shouldn't really distribute more than one CR as an example, but that change can wait till later.

Reply:
done